Tasks/SonarQubePostTest/SummaryReport/DashboardUriSection.ps1

```diff
@@ -0,0 +1,22 @@
+function GetDashboardUriSectionContent
```

Comment:
If I were using a normal language like C# I would have used some aggregator sort of pattern. But this is PS , so it will have to be done by convention - the top level report builder calls each Get_Section function to construct the report and then combines the sections.